libr/core/cmd_print.c

```diff
@@ -2116,10 +2145,19 @@ static int cmd_print(void *data, const char *input) {
 				" esil (pae) from hexpairs\n");
 		} else {
 			RAsmCode *acode;
+			int i;
+			int bytes;
 			r_asm_set_pc (core->assembler, core->offset);
 			acode = r_asm_massemble (core->assembler, input+1);
 			if (acode && *acode->buf_hex) {
-				r_cons_printf ("%s\n", acode->buf_hex);
+				bytes = strlen(acode->buf_hex) >> 1;
+				for (i = 0; i < bytes; i++) {
+					if (!core->print->big_endian)
```

Comment:
wrong spaces before parenthesis, negated conditionals by default makes hard to read and spaguetti